parlai/crowdsourcing/tasks/turn_annotations_static/turn_annotations_blueprint.py

```diff
@@ -215,15 +226,28 @@ def process_data(self, data_dicts, annotation_indices=None):
                     )
                 for a in annotation_indices[conv_idx]:
                     processed_dialog = self._process_conversation(d, [a])
+                    total_annotation_count += 1
```

Comment:
hmm - won't this double-count given the `total_annotation_count += len(annotation_indices[conv_idx])` line above?